components/dashboard/src/admin/TeamsSearch.tsx

```diff
@@ -0,0 +1,138 @@
+        setSearching(true);
+        try {
+            const result = await getGitpodService().server.adminGetTeams({
+                searchTerm,
+                limit: 100,
+                orderBy: 'creationTime',
+                offset: 0,
+                orderDir: "desc"
+            })
+            setSearchResult(result);
+        } finally {
+            setSearching(false);
+        }
+    }
+    return <>
+        <div className="pt-8 flex">
+            <div className="flex justify-between w-full">
+                <div className="flex">
+                    <div className="py-4">
+                        <svg className={searching ? 'animate-spin' : ''} width="16" height="16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path fillRule="evenodd" clipRule="evenodd" d="M6 2a4 4 0 100 8 4 4 0 000-8zM0 6a6 6 0 1110.89 3.477l4.817 4.816a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 010 6z" fill="#A8A29E" />
+                        </svg>
+                    </div>
+                    <input type="search" placeholder="Search Teams" onKeyDown={(k) => k.key === 'Enter' && search()} onChange={(v) => { setSearchTerm(v.target.value) }} />
+                </div>
+                <button disabled={searching} onClick={search}>Search</button>
+            </div>
+        </div>
+        <div className="flex flex-col space-y-2">
+            <div className="px-6 py-3 flex justify-between text-sm text-gray-400 border-t border-b border-gray-200 dark:border-gray-800 mb-2">
+                <div className="w-7/12">Name</div>
+                <div className="w-5/12 flex items-center"><span>Created</span>
+                    <svg xmlns="http://www.w3.org/2000/svg" fill="none" className="h-4 w-4" viewBox="0 0 16 16"><path fill="#A8A29E" fill-rule="evenodd" d="M13.366 8.234a.8.8 0 010 1.132l-4.8 4.8a.8.8 0 01-1.132 0l-4.8-4.8a.8.8 0 111.132-1.132L7.2 11.67V2.4a.8.8 0 111.6 0v9.269l3.434-3.435a.8.8 0 011.132 0z" clip-rule="evenodd" /></svg>
+                </div>
+
+
+            </div>
+            {searchResult.rows.map(team => <TeamResultItem team={team} members={teamMembers[team.id]} />)}
+        </div>
+    </>
+
+function TeamResultItem(props: { team: Team, members: TeamMemberInfo[]}) {
+    const { team, members } = props;
+    return (
+        <Link key={'pr-' + team.name} to={'/admin/teams/' + team.id} data-analytics='{"button_type":"sidebar_menu"}'>
+            <div className="rounded-xl whitespace-nowrap flex py-6 px-6 w-full justify-between hover:bg-gray-100 dark:hover:bg-gray-800 focus:bg-gitpod-kumquat-light group">
+                <div className="flex flex-col w-7/12 truncate">
+                    <div className="font-medium text-gray-800 dark:text-gray-100 truncate">{team.name}
+                    {team.markedDeleted && <Label text='Deleted' color="red"/>}
+                    {members?.length > 0 && <p>{`${members?.length} ${members?.length > 1 ? `members` : `member`}`}</p>}
```

Comment:
**thought**: I still think the members count information is valuable but the current design could be slightly improved. What do you think of re-using the tabular pattern from projects, users, and workspaces here even if this is just a single column information?

| BEFORE | AFTER |
|-|-|
| <img width="805" alt="members-before" src="https://user-images.githubusercontent.com/120486/155566647-141e1b87-09b5-4126-a35b-d8d4331ca882.png"> | <img width="805" alt="members-after" src="https://user-images.githubusercontent.com/120486/155566652-88a56105-9123-4770-83f4-3427fa35d18f.png"> |